charts/datadog-operator/templates/clusterrole.yaml

```diff
@@ -6,70 +6,275 @@ metadata:
   labels:
 {{ include "datadog-operator.labels" . | indent 4 }}
 rules:
+- nonResourceURLs:
+  - /metrics
+  verbs:
+  - get
 - apiGroups:
-  - "security.openshift.io"
+  - ""
   resources:
-  - securitycontextconstraints
+  - componentstatuses
   verbs:
-  - use
-  resourceNames:
-  - restricted
```

Comment:
why do we remove it?

Reply:
I used the list generated by kubebuilder from annotations to align Kustomize and Helm chart. We should add it there so that it gets generated

Reply:
Fixed